### PR TITLE
Add the Create Table API to JS-SDK

### PR DESCRIPTION
**Pull request by viktor-liablin**
Add the Create Table API to JS-SDK

### CODE REVIEW

File: src/management/data.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    this.app = app
+  }
+
+  createTable(name, columns) {
+    if (!name || typeof name !== 'string') {
+      throw new Error('Table name must be provided and must be a string.')
+    }
+
+    if (!Array.isArray(columns)) {
+      throw new Error('Columns must be a list.')
```
Comment (stanislav-grin) [inline src/management/data.js:12]: The same here

File: test/unit/specs/management/data.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+      const result = await Backendless.Management.Data.createTable('MyTable', [])
+
+      expect(request).to.deep.include({
+        method: 'POST',
+        path  : `${APP_PATH}/management/data/table`,
+        body  : { name: 'MyTable', columns: [] }
+      })
+
+      expect(result).to.be.eql(fakeResult)
```
[the remark this answers is not in the document]
Comment (Valodya) [reply test/unit/specs/management/data.js:23]: I assume it should return a table object, at least there will be an `id`

[the remark this answers is not in the document]
Comment (viktor-liablin) [reply test/unit/specs/management/data.js:23]: https://monosnap.com/file/oiMCgA8SBCxyXGm015EfRxglVQnnCl

[the remark this answers is not in the document]
Comment (viktor-liablin) [reply test/unit/specs/management/data.js:23]: nothing is returned by this request

**Pull request merged**